packages/client/eslint.config.js

```diff
@@ -1,11 +1,17 @@
+import { fixupPluginRules } from "@eslint/compat"
 import js from "@eslint/js"
+import pluginVitest from "@vitest/eslint-plugin"
```

Comment:
adds vitest and playwright plugins, applied only in the restricted config blocks towards the end of the config